tests/data/original/test.conf

```diff
@@ -0,0 +1,149 @@
+{
+  "licenseDirectoryPath" : "/scratch/PELE-repo/licenses",
+  
+  "Initialization" : {
+     "allowMissingTerminals": true,
+     "ForceField" : "OPLS2005",
+     "MultipleComplex" : [ {"files" : [{"path": "/home/dsoler/repos/frag_pele/tests/1w7h_preparation_structure_2w_carbonylC6-H7O1-C1/clustering_PDBs/1/initial_0_0.pdb" }] },
+{"files" : [{"path": "/home/dsoler/repos/frag_pele/tests/1w7h_preparation_structure_2w_carbonylC6-H7O1-C1/clustering_PDBs/1/initial_0_1.pdb" }] },
```

Comment:
Take in mind that the paths of the PDBS are from your computer.

Reply:
True, true, I'll have a look at how to test the restart in a more general manner.

Reply:
The last test has been done more generally by using a templetized control file in ffad675f3653256d38ea8fc96a62d43164698872